.github/workflows/a-team.yml

```diff
@@ -99,3 +99,11 @@ jobs:
           scp -o StrictHostKeyChecking=no -i ~/.ssh/id_rsa tests/ci/fix_baremetal.sh ${{ secrets.HOST }}:
           ssh -o StrictHostKeyChecking=no -i ~/.ssh/id_rsa ${{ secrets.HOST }} ./fix_baremetal.sh
         shell: bash
+
+      - name: Archive production artifacts
+        uses: actions/upload-artifact@v2
+        with:
+          name: tmt run logs without image
+          path: |
+            /var/run/tmt/$GITHUB_RUN_ID
```

Comment:
Where are the logs going to be archived? Is it local to the BM machine? If yes, it would be valuable to push them to a persistent storage. Or, use AWS logging service.

Reply:
no idea currently, I thought gitlab provides something :) Checking it ;)

Reply:
Yes, the logs for the tmt runs are getting stored right now locally at the BM machine.
But once we use Testing Farm, the logs will be stored there.

Reply:
+1